src/include/catalog/pg_proc.h

```diff
@@ -5768,6 +5768,11 @@ DESCR("filtered space saving weighted increment");
 DATA(insert OID = 4417 ( pipeline_exec_adhoc_query PGNSP PGUID 12 1 0 0 0 f f f f t f s 1 0 25 "25" _null_ _null_ _null_ _null_ _null_ pipeline_exec_adhoc_query _null_ _null_ _null_ ));
 DESCR("exec adhoc query");
 
+//DATA(insert OID = 230 (  dsqrt			   PGNSP PGUID 12 1 0 0 0 f f f f t f i 1 0 701 "701" _null_ _null_ _null_ _null_ _null_	dsqrt _null_ _null_ _null_ ));
+// provolatile
+DATA(insert OID = 4418 ( pipeline_send_alert_new_row PGNSP PGUID 12 1 0 0 0 f f f f t f v 0 0 2279 "" _null_ _null_ _null_ _null_ _null_ pipeline_send_alert_new_row _null_ _null_ _null_ ));
```

Comment:
pipeline_send_alert_new_row is the trigger function that enables sending alerts from a trigger, it currently needs to be public to work with the current syntax

e.g:

```
create continuous view v as select x::text,count(*) from s group by x;

create trigger trigg after update or insert on v for row 
WHEN (true)
execute procedure pipeline_send_alert_new_row();
```